addons/website/static/src/snippets/s_popup/options.js

```diff
@@ -87,6 +87,15 @@ options.registry.SnippetPopup = options.Class.extend({
         const color = widgetValue ? 'var(--black-50)' : '';
         this.$target[0].style.setProperty('background-color', color, 'important');
     },
+    /**
+     * @see this.selectClass for parameters
+     */
+    addAnchor(previewMode, widgetValue, params) {
+        if (widgetValue) {
+            const widget = this._requestUserValueWidgets('anchor_opt')[0];
+            widget.el.click(); // TODO: to find why widget.enable() doesnt work here
```

Comment:
TODO? Should be easy enough to debug I guess?

Reply:
no longer needed